meta-xt-prod-aos-rcar-domd/recipes-aos/aos-updatemanager/aos-updatemanager_git.bbappend

```diff
@@ -0,0 +1,78 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
Maybe "[domd] Add UM" can be squashed with the previous commit "[domf] Add UM".

Reply:
I'll keep as separate. To copy have possibility separately cherry-pick commits to another products for example.